### PR TITLE
Screens

### PR DESCRIPTION
**Pull request by tanveertkd**
This PR has the following screens  

- Home
- Categories
- Single Quiz Page
- Login

### CODE REVIEW

**Review by poornima-alugubelly — APPROVED** [review]
Everything looks good

File: src/components/Authentication/Login.tsx [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+    return (
+        <div className="right login-container">
+            <div className="login">
+                <p className="login-header">Sign In</p>
+
+                <form className="login-input-form">
+                    <div className="form-input">
+                        <label htmlFor="login-email" className="input-label">
+                            {' '}
+                            Email:{' '}
```
Comment (msdeshmukh009) [inline src/components/Authentication/Login.tsx:14]: You can remove these comments which were added by IDE.

File: src/components/Quiz/Quiz.tsx [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+            // {
+            //     question: "Question 2",
+            //     options: [
+            //         'This is option 1',
+            //         'This is option 2',
+            //         'This is option 3',
+            //         'This is option 4',
+            //     ],
+            //     answer: "option2"
+            // },
```
Comment (msdeshmukh009) [inline src/components/Quiz/Quiz.tsx:27]: I guess you forgot to remove this comment.

**Review by msdeshmukh009 — APPROVED** [review]
The rest is looking good.

File: src/components/Quiz/Quiz.tsx (hunk 2 above)
Comment (tanveertkd) [reply src/components/Quiz/Quiz.tsx:27]: yes, this was just to test something

**Pull request merged**